README.md

```diff
@@ -1,7 +1,39 @@
 # gainbridge-qa
 
+## Installation
+
+1. Clone the repository:
+
+   ```bash
+
+   git clone "repo url"
+   cd repository
+
+2. Install dependencies:
+
+    npm install
+
+3. So that all dependencies which are present in package.json will be installed in your system
+
+# Cypress-Automation
+
+Cypress is a next generation front end testing tool built for the modern web.
+
+This project aims to automate test cases for verifying UI elements and validating functionality across a range of input values.
+
 ## Cypress Automation Execution Procedure ?
 
+# To run:-
+1. Once all dependencies are installed
+
+2. Run the following command which will open Cypress dashboard
+```
+npx cypress run
+```
+3. Above command will execute the scripts
```

Comment:
keep commands to run from CMD and also let them know where are the reports can be stored

Reply:
> keep commands to run from CMD and also let them know where are the reports can be stored
Sir, I have updated as per review comments 

![2 gainBridge](https://github.com/SureshQAExpert/gainbridge-qa/assets/168629072/9fd95ce0-44fb-4044-8d16-c04b627a467e)